psp_notes.md

```diff
@@ -55,8 +55,9 @@ https://github.com/api3dao/airnode/blob/v1-protocol/packages/airnode-node/src/ev
 
 5. The Airnode makes the API call specified by the template and the additional parameters
 
-Probably current [call-api-ts](https://github.com/api3dao/airkeeper/blob/main/src/call-api.ts) in Airkeeper can be
-re-used for this step.
+Probably current
+[call-api-ts](https://github.com/api3dao/airkeeper/blob/2de24dd82d710d1dc383049f2637a57b15d90ae2/src/call-api.ts) in
```

Comment:
Since I've merge the changes from the refactor then we can keep pointing to main branch: 

```suggestion
[call-api-ts](https://github.com/api3dao/airkeeper/blob/main/src/api/call-api.ts) in
```

Reply:
Awesome thanks :)